src/util/Util.js

```diff
@@ -113,6 +147,36 @@ define(['okta', 'util/Logger'], function (Okta, Logger) {
     controller.trigger('afterError', { controller: className }, error);
   };
 
+  /**
+   * Why redirect via Form get rather using `window.location.href`?
+   * At the time of writing, Chrome (<72) in Android would block window location change
+   * at following case
+   * 1. An AJAX is trigger because of user action.
+   * 2. 5+ seconds passed without any further user interaction.
+   * 3. User takes an action results in window location change.
+   *
+   * Luckily we discovered that uses Form submit would work around this problem
+   * even though it changed window location.
+   *
+   * Check the commit history for more details.
+   */
+  Util.redirectWithFormGet = function (url) {
+    if (!url) {
+      Logger.error(`Cannot redirect to empty URL: (${url})`);
+      return;
+    }
+
+    var mainContainer = document.getElementById(Enums.WIDGET_CONTAINER_ID);
+    if (!mainContainer) {
+      Logger.error('Cannot find okta-sign-in container append to which a form');
```

Comment:
If we hit this conditional, the user will not be redirected to Okta and/or a defined step-up URL.Any reason **not** to use the "global error" handler, as the Widget cannot recover from this state?

Reply:
actually good question!
My initial thought was those 2 error are more for programming safety rather runtime exception (user has nothing to do with cause nor fix it) hence I don't even think of `callGlobalErrorFn`.
hmm, turns of cause some inconsistency.

Reply:
Agreed - I think your decision to use the `Logger` is correct, we're just a bit inconsistent :)

Alternatively, we can also trigger an `afterError` event, so developers can respond accordingly.

This might require a bit more investigation, so what do you think about adding it to [OKTA-200220](https://oktainc.atlassian.net/browse/OKTA-200220)?

Reply:
I wouldn't think it's worth to be a `afterError` event and more prefer `callGlobalError` if do want to bubble up the error.

Reply:
I'm debating myself whether it's good idea to trigger an event or failed silently (log to console) to developer.
Didn't have a good answer hence add comment to the ticket.
Thanks for raising the question!

Reply:
Isn't this an error that should never happen? Since the Widget is in control of `WIDGET_CONTAINER_ID`. If this check fails, then something is really wrong and the Widget is likely not even in the DOM.

There's no need to emit `afterError` because there's nothing a developer could do. JavaScript console makes sense to me.